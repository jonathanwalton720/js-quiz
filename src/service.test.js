import service from './service';

test('it returns correct mock data', () => {
  var quizQuestions = service.GetQuizQuestions();
  expect(quizQuestions).toMatchSnapshot();
})
