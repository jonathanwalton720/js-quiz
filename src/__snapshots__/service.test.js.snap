// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it returns correct mock data 1`] = `
Array [
  QuizQuestion {
    "choices": Array [
      "David Cameron",
      "Gordon Brown",
      "Winston Churchill",
      "Tony Blair",
    ],
    "correctAnswer": 0,
    "question": "Who is Prime Minister of the United Kingdom?",
  },
  QuizQuestion {
    "choices": Array [
      "10 days",
      "12 days",
      "4 days",
      "8 days",
    ],
    "correctAnswer": 0,
    "question": "If a snail climbed up a 12 ft wall at a steady rate of 3 ft per day, but slipped down 2 ft every night, how many days would it take him to reach the top?",
  },
  QuizQuestion {
    "choices": Array [
      "Sixteen",
      "Twelve",
      "Four",
      "Fifteen",
    ],
    "correctAnswer": 0,
    "question": "On his egg deliveries, farmer Fred sets off with 400 eggs. At the first delivery he drops off 5 doz\`, at the second 12 doz\`, and at the last one he delivers 15 doz\`. How many eggs does he take home to his brooding wife?",
  },
  QuizQuestion {
    "choices": Array [
      "Track 29",
      "Track 37",
      "Track 22",
      "Track 13",
    ],
    "correctAnswer": 0,
    "question": "According to the song, from which track does the Chattanooga choo choo leave Pennsylvania station?",
  },
  QuizQuestion {
    "choices": Array [
      "Deputy Dawg",
      "Mighty Mouse",
      "Dinky Duck",
    ],
    "correctAnswer": 0,
    "question": "Who do Vince and Musky accompany in their cartoon capers?",
  },
  QuizQuestion {
    "choices": Array [
      "Jenny",
      "Sow",
      "Vixen",
      "Mare",
    ],
    "correctAnswer": 0,
    "question": "What girls name is also the term used to describe a female donkey?",
  },
  QuizQuestion {
    "choices": Array [
      "Harold",
      "Alfred",
      "Edward",
      "Cathute",
    ],
    "correctAnswer": 0,
    "question": "Name the king who failed to keep an eye on things at the battle of Hastings?",
  },
  QuizQuestion {
    "choices": Array [
      "Walnut",
      "Mahogany",
      "Oak",
      "Teak",
    ],
    "correctAnswer": 0,
    "question": "Traditionally, what type of wood do Rolls Royce use on the dashboards of their cars?",
  },
  QuizQuestion {
    "choices": Array [
      "Polo",
      "Cricket",
      "Golf",
    ],
    "correctAnswer": 0,
    "question": "In which sport would you use a chucker?",
  },
  QuizQuestion {
    "choices": Array [
      "Knitting",
      "Shucking Oysters",
      "Surfing",
      "Skydiving",
    ],
    "correctAnswer": 0,
    "question": "If you happened to be plaining and pearling what would you be doing?",
  },
  QuizQuestion {
    "choices": Array [
      "Maple Leaf",
      "Sycamore Leaf",
      "Fig Leaf",
    ],
    "correctAnswer": 0,
    "question": "What is the national emblem of Canada?",
  },
  QuizQuestion {
    "choices": Array [
      "Carnivores",
      "Omnivores",
      "Herbivores",
    ],
    "correctAnswer": 2,
    "question": "Are Gorillas carnivores, omnivores, or herbivores?",
  },
  QuizQuestion {
    "choices": Array [
      "A whale",
      "A duck",
      "A labrador",
    ],
    "correctAnswer": 0,
    "question": "Free Willey was a film about what?",
  },
  QuizQuestion {
    "choices": Array [
      "Yes",
      "No",
    ],
    "correctAnswer": 0,
    "question": "Can pigs swim? Yes or no.",
  },
  QuizQuestion {
    "choices": Array [
      "Evacuation of Dunkirk",
      "First Arctic Convoy to the Soviet Union",
      "Anti-shipping operation off Norway",
    ],
    "correctAnswer": 0,
    "question": "What World War II operation was code named \`Dynamo\`?",
  },
  QuizQuestion {
    "choices": Array [
      "Four",
      "Twenty-Five",
      "Ten",
    ],
    "correctAnswer": 0,
    "question": "How many old pennies were there in a Groat?",
  },
  QuizQuestion {
    "choices": Array [
      "Feed it to his animals",
      "Mix it with his fertilizer",
    ],
    "correctAnswer": 0,
    "question": "What would a farmer do with his mangelwurzel?",
  },
  QuizQuestion {
    "choices": Array [
      "Laying",
      "Leaping",
      "Honking",
    ],
    "correctAnswer": 0,
    "question": "In the song \`The 12 days of Christmas\`, what are the 6 geese a-doing?",
  },
  QuizQuestion {
    "choices": Array [
      "Between the wickets.",
      "Surrounding the boundary.",
      "Measuring the pitch.",
    ],
    "correctAnswer": 0,
    "question": "In cricket, where would you find the chain?",
  },
]
`;
